packages/rum/src/domain/record/record.spec.ts

```diff
@@ -334,20 +335,26 @@ describe('record', () => {
       expect(inputRecords.length).toBe(1)
     })
 
-    it('should record the scroll event inside a shadow root', () => {
-      const div = appendElement('<div></div>', createShadow()) as HTMLDivElement
+    fit('should record the scroll event inside a shadow root', () => {
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏😬